### PR TITLE
Added autocomplete: off to the title text field

### PR DESCRIPTION
**Pull request by pmckernin**

### CODE REVIEW

**Comment by pmckernin** [comment]
I have tested this and it appears that I have successfully turned off the autocomplete. 

To test, paste the following code into your gemfile in place of the current web_git

` gem 'web_git', github: 'firstdraft/web_git', branch: "c262ede0f7ceac2c8b1e00532b449e4a446140dd"`

File: app/views/web_git/commands/status.html.erb [hunk 1]
```diff
@@ -44,9 +44,8 @@
                   <%= form_tag commits_url do %>
                     <div class="form-group">
                       <%= label_tag :title %>
-                      <%= text_field_tag :title, nil, placeholder: "Enter a title for your commit (required)...", class: "form-control", required: true, autofocus: true %>
+                      <%= text_field_tag :title, nil, placeholder: "Enter a title for your commit (required)...", class: "form-control", required: true, autofocus: true, html: { autocomplete: "off" } %>
```
Comment (jelaniwoods) [inline app/views/web_git/commands/status.html.erb]: I don't think this is rendering the right html. It creates this:
patrick-off-autocomplete

The html should render kind of [like this](https://developer.mozilla.org/en-US/docs/Web/Security/Securing_your_site/Turning_off_form_autocompletion#Disabling_autocompletion) from what I understand. Although, there appears to be [some debate](https://stackoverflow.com/a/51686510) over how browsers choose to follow this setting.

According to [this answer](https://stackoverflow.com/a/24781882), it looks like that's how you disable autocomplete on the `form_tag` method but not a `text_field_tag`.

**Comment by jelaniwoods** [comment]
@pmckernin LGTM 🚀

**Comment by jelaniwoods** [comment]
Looks like we need to do this again for the Branch form.

**Pull request merged**